dns/r53/client.go

```diff
@@ -0,0 +1,67 @@
+package r53
+
+import (
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+)
+
+const (
+	hostedZoneID = "Z2KHQXXHNXGTXW"
+)
+
+// Route53Client enables interaction with aws route53
+type Route53Client struct {
+	client route53.Route53
+}
+
+// New creates a route53 client used to interact with aws
+func New() Route53Client {
+	return Route53Client{
+		client: *route53.New(session.New(), &aws.Config{Region: aws.String("eu-west-1")}),
+	}
+}
+
+// UpdateRecordSets updates records in aws based on the change list.
+func (dns *Route53Client) UpdateRecordSets(changes []*route53.Change) error {
+
+	recordSetsInput := &route53.ChangeResourceRecordSetsInput{
+		HostedZoneId: aws.String(hostedZoneID),
+		ChangeBatch: &route53.ChangeBatch{
+			Changes: changes,
+		},
+	}
```

Comment:
100

I will add it to the story to deal with a greater number and do in a different PR as this PR is humongous